packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -92,6 +92,11 @@ export class VaModal {
    */
   @Prop() disableAnalytics?: boolean = false;
 
+  /**
+   * If `true`, modal will be wider.
+   */
+  @Prop({ reflect: true }) large?: boolean = false;
```

Comment:
I'm not sure how I feel about this prop. When I tried it in the Storybook canvas, I did not see any change to the modal when I toggled it. Have you talked with Natalie about it? Looking at our current modal implementation and the modals in Storybook it seems to me we need to have a larger conversation about modal widths. Also, if we're going to keep it, I we should add a story to storybook and design.va.gov to show it.

Reply:
Good catch. I forgot to update the `va-modal` stories to include the new `large` prop. This change was required to match the existing modal style used in the OMBInfo React component.

Without this prop:
<img width="1553" alt="Screen Shot 2022-07-27 at 09 18 11" src="https://user-images.githubusercontent.com/36863582/181517808-82970b1e-7ce1-444c-a427-ed146dc8883d.png">

With this prop:
<img width="1559" alt="Screen Shot 2022-07-27 at 09 26 51" src="https://user-images.githubusercontent.com/36863582/181517849-cfd3cb0f-af3e-44e7-8cd1-f51d2b234639.png">

OMBInfo React component:
<img width="1548" alt="Screen Shot 2022-07-28 at 09 34 34" src="https://user-images.githubusercontent.com/36863582/181518349-a829f6d6-cc4f-4a2a-a862-b8d86b224307.png">

Reply:
Per standup: Reach out to Natalie for Modal design work

Reply:
Looks good in Storybook, thank you for adding the story. Once this is merged, please add this story to design.va.gov as well.